frontend/src/components/ElectionForm/ElectionDetails.tsx

```diff
@@ -54,10 +69,6 @@ export default function ElectionDetails({ election, applyElectionUpdate, getStyl
                 newErrors.startTime = 'Invalid date';
                 isValid = 0;
             }
-            else if (election.start_time < new Date()) {
-                newErrors.startTime = 'Start date must be in the future';
```

Comment:
I'm curious why this was removed, did we identity scenarios where users would want to set a start date in the past?

Reply:
I wasn't sure if this really helps anything. One thing I wanted with the enable checkbox was to auto-populate with default times so its clear what needs to be entered. The current time seemed like the least annoying option for people that just want to set an end time, but that would potentially trip this error. Worst case the election just starts immediately.